e2e-tests/insurance/cypress/e2e/journeys/your-business/turnover/turnover-financial-year-end.spec.js

```diff
@@ -157,4 +160,17 @@ context(`Insurance - Your business - Turnover page - submitting a company with $
     field.hint().should('not.exist');
     field.label().should('not.exist');
   });
+
+  it(`should display ${FINANCIAL_YEAR_END_DATE} as "${DEFAULT.EMPTY}" in summary list`, () => {
```

Comment:
The description here and in the previous tests are confusing, 

Should we be testing the summary list in another test?

Reply:
made new test